### DISCUSSION TITLE
Last Updated never updates

### DISCUSSION THREAD

**Discussion by audioscavenger** [allinurl/goaccess#2664] [Q&A]
It's maybe a dumb question but i could not find any answer in the man pages.
How do we get the Last Updated date to update? What is the purpose of `--real-time-html` then?

**Comment by allinurl** [comment]
Are you still facing trouble with this? Have you looked at your browser's console to see if there's any error showing up? If your setup is correct, the data should update in real-time as it arrives in the log.
